release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -372,6 +372,15 @@ Pods created using the macvlan CNI plugin, where the IP address management CNI p
 [id="ocp-4-15-registry"]
 === Registry
 
+[id="ocp-4-15-private-storage-endpoint-azure"]
+==== Support for private storage endpoint on {azure-short}
+
+With this release, the Image Registry Operator can be leveraged to use private storage endpoints on {azure-short}. This feature enables seamless configuration of private endpoints for storage accounts when {product-title} is deployed on private {azure-short} clusters, which allows users to deploy the image registry without exposing public-facing storage endpoints.
+
+See xref:../post_installation_configuration/configuring-private-cluster.adoc#registry-configuring-private-storage-endpoint-azure_configuring-private-cluster[Configuring a private storage endpoint on Azure] for more information.
+
+For more information about installing a private image registry on a private Microsoft Azure cluster, see xref:../installing/installing_azure/installing-azure-private.adoc#installing-private-image-registry-private-azure[Optional: Preparing a private Microsoft Azure cluster for a private image registry].
```

Comment:
Suggest maybe add an Additional resources section for the link since it's two at the end close together like this?